lightning/src/routing/scoring.rs

```diff
@@ -727,9 +727,16 @@ impl<L: Deref<Target = u64>, T: Time, U: Deref<Target = T>> DirectedChannelLiqui
 		} else {
 			let numerator = (max_liquidity_msat - amount_msat).saturating_add(1);
 			let denominator = (max_liquidity_msat - min_liquidity_msat).saturating_add(1);
-			let negative_log10_times_2048 =
-				approx::negative_log10_times_2048(numerator, denominator);
-			self.combined_penalty_msat(amount_msat, negative_log10_times_2048, params)
+			if amount_msat - min_liquidity_msat < denominator / 64 {
```

Comment:
Is `64` the same as `LOWER_BITS_BOUND`?

Reply:
Hmm, uhmmmmm, I guess? I'll go ahead and change it to reference the constant cause they're tightly tied, but it was more of an experimental thing. Honestly we should probably do `LOWER_BITS_BOUND / 2` here but you end up hitting the cutoff before the penalty gets close to 500, creating a new non-linearity there.